src/usecases/deleteVisitByCallId.js

```diff
@@ -0,0 +1,28 @@
+const deleteVisitByCallId = ({ getDb }) => async (callId) => {
+  const db = await getDb();
+  console.log("deleting visit for  ", callId);
+  try {
+    const results = await db.any(
+      `DELETE FROM scheduled_calls_table WHERE call_id = $1 LIMIT 1`,
+      callId
+    );
+
+    const result = results[0];
+    console.log("result", result);
+
+    return {
+      result: {
+        success: true,
+      },
+      error: null,
+    };
```

Comment:
What do you think of the below? I don't think the `result` key brings much value and simplifies the check that the usecase has successfully deleted a visit.

```suggestion
    return {
      success: true,
      error: null,
    };
```

Reply:
Agree :)